orion.py

```diff
@@ -67,7 +67,7 @@ def orion(**kwargs):
         benchmarkIndex=test['benchmarkIndex']
         uuid = kwargs["uuid"]
         baseline = kwargs["baseline"]
-        match = Matcher(index="ospst-perf-scale-ci-*",
+        match = Matcher(index="ospst-perf-scale-ci*",
```

Comment:
this was added to the config files themselves. We need to update to get the index from the file